src/pages/DefaultOrgSelector/DefaultOrgSelector.jsx

```diff
@@ -0,0 +1,126 @@
+import { zodResolver } from '@hookform/resolvers/zod'
+import { Controller, useForm } from 'react-hook-form'
+import { Redirect, useHistory, useParams } from 'react-router-dom'
+import { z } from 'zod'
+
+import { useUpdateDefaultOrganization } from 'services/defaultOrganization'
+import { trackSegmentEvent } from 'services/tracking/segment'
+import { useUser } from 'services/user'
+import { mapEdges } from 'shared/utils/graphql'
+import Avatar from 'ui/Avatar/Avatar'
+import Button from 'ui/Button'
+import Select from 'ui/Select'
+
+import GitHubHelpBanner from './GitHubHelpBanner'
+
+import { useMyOrganizations } from '../TermsOfService/hooks/useMyOrganizations'
+
+const FormSchema = z.object({
+  select: z.string().nullish(),
+})
+
+const renderItem = ({ item }) => {
+  if (!item) return null
+
+  return (
+    <div className="flex h-8 items-center gap-2">
+      {item?.avatarUrl && <Avatar user={item} />}
+      <span>{item?.username}</span>
+    </div>
+  )
+}
+
+function DefaultOrgSelector() {
+  const { register, control, setValue, handleSubmit } = useForm({
+    resolver: zodResolver(FormSchema),
+    mode: 'onChange',
+  })
```

Comment:
Can you pull in the form state here and get the errors. When I don't have an org selected I can click `Continue to app` but nothing happens. We should display and error when they try, we should probably also disable the button until they select an org.

Reply:
we don't block access to app if user doesn't select an org, they land on self org

Reply:
Fixed with [ad34792](https://github.com/codecov/gazebo/pull/2160/commits/ad34792b588c3d616a2a35f3149268c29d18c07b)